extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -0,0 +1,1691 @@
+
+### Feature Textures
+
+Feature textures (not to be confused with [Feature ID Textures](#feature-id-texture)) use textures rather than parallel arrays to store values. Feature textures are accessed directly by texture coordinates, rather than feature IDs. Feature textures are especially useful when texture mapping high frequency data to less detailed 3D surfaces.
+
+Feature textures use the [Raster Format](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#raster-format) of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata) with a few additional constraints:
+
+* A scalar property cannot be encoded into multiple channels. For example, it is not possible to encode a `UINT32` property in an `RGBA8` texture.
+* Components of fixed-length array properties must be separate channels within the same texture.
+* Variable-length arrays are not supported.
+
+Additionally, the data type and bit depth of the image must be compatible with the property type. An 8-bit per pixel RGB image is only compatible with `UINT8` or normalized `UINT8` properties, and array properties thereof with three components or less. Likewise, a floating point property requires a floating point-compatible image format like KTX2 which may require additional extensions.
+
+Feature textures are defined with the following steps:
+
+1. A class is defined in the root `EXT_feature_metadata` extension object. This is used to describe the metadata in the texture.
+2. A feature texture is defined in the root `EXT_feature_metadata.featureTextures` object. This must reference the class ID defined in step 1.
+3. A feature texture is associated with a primitive by listing the feature texture ID in the `primitive.EXT_feature_metadata.featureTextures` array.
+<img src="figures/feature-texture.png"  alt="Feature Texture" width="500">
+
+_Class and feature texture_
+
+```jsonc
+{
+  "extensions": {
+    "EXT_feature_metadata": {
+      "schema": {
+        "classes": {
+          "heatSample": {
+            "properties": {
+              "heatSample": {
+                "type": "UINT8",
+                "normalized": true
+              }
+            }
+          }
+        }
+      },
+      "featureTextures": {
+        "heatLossTexture": {
+          "class": "heatSample",
+          "properties": {
+            "heatLoss": {
+              "texture": {
+                "index": 0,
+                "texCoord": 0
+              },
+              "channels": "r"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+_Primitive_
+
+```jsonc
+{
+  "primitives": [
+    {
+      "attributes": {
+        "POSITION": 0,
+        "TEXCOORD_0": 1
+      },
+      "indices": 2,
+      "material": 0,
+      "extensions": {
+        "EXT_feature_metadata": {
+          "featureTextures": ["heatLossTexture"]
```

Comment:
See above — depending on that answer there is some ambiguity here as to what it means to specify multiple feature textures in this primitive list.

Is it correct to say that different primitives could represent features of the same class, but use different feature textures?

Reply:
>Is it correct to say that different primitives could represent features of the same class, but use different feature textures?

Yes, this would be similar to how two primitives could use the same class but use different feature tables. If there's a way to make both of these clearer, that would be helpful. I've certainly had this confusion about feature tables in the past.

Reply:
Resolving in https://github.com/CesiumGS/glTF/pull/14.